app/components/header/HeaderSubpage.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import Image from 'next/image'; 
+
+const HeaderSubpage: React.FC = () => {
+    return (
+        <header className="flex justify-between items-center bg-[#000582] p-4 text-white relative">
```

Comment:
bg-[#000582] Please take a look at the CSS name convention which we defined
Is relative needed here?